README.md

```diff
@@ -36,6 +36,33 @@ To see the graphics, you can open another terminal and run
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
 
+## Environments and Tasks
```

Comment:
Introduce gym interfaces and training examples.

Reply:
Added training examples part. 
Does "gym interfaces" mean the detailed observation, action and rewards? Maybe people can refer to comments in source files for them.

Reply:
What I meant is add a sentence saying that we provide OpenAI gym interfaces so that people can easily integrate them into their RL training algorithms.